zerver/lib/actions.py

```diff
@@ -4406,6 +4411,60 @@ def do_update_outgoing_webhook_service(bot_profile, service_interface, service_p
                     ),
                bot_owner_user_ids(bot_profile))
 
+def do_update_bot_config_data(bot_profile: UserProfile,
+                              config_data: Dict[Text, Text]) -> None:
+    for key, value in config_data.items():
+        set_bot_config(bot_profile, key, value)
+    send_event(dict(type='realm_bot',
+                    op='update',
+                    bot=dict(email=bot_profile.email,
+                             user_id=bot_profile.id,
+                             services = [dict(config_data=config_data)],
```

Comment:
So, if one changed only a few pieces of a bot's config data, do we want this to send them all or just the ones that changed?